schemadiff/validation_rules/__init__.py

```diff
@@ -66,7 +66,8 @@ def is_valid(self) -> bool:
 
     @property
     def message(self):
-        return f"{self.change.message} without a description for the type or one of its fields (rule: `{self.name}`)"
+        return f"{self.change.message} without a description for the type or one of its fields (rule: `{self.name}`). " \
```

Comment:
We had this `for the type or one of its fields` because we didn't have the path. Now that we do we could write something like:
`without a description for {self.change.path} (rule: `{self.name}`). "`